xml/System.IO/WindowsRuntimeStorageExtensions.xml

```diff
@@ -52,6 +52,28 @@ The next example shows the XAML code that is associated with the previous exampl
     </remarks>
   </Docs>
   <Members>
+    <MemberGroup MemberName="CreateSafeFileHandle">
+      <AssemblyInfo>
+        <AssemblyName>System.Runtime.WindowsRuntime</AssemblyName>
+        <AssemblyVersion>4.0.14.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Docs>
+        <summary>Creates and returns a safe file handle for the IStorage instance that is being extended.</summary>
+        <remarks>
+          <format type="text/markdown">
+            <![CDATA[
+
```

Comment:
The key thing here is that you can get a brokered SafeFileHandle. This is critically important if you want to create a FileStream around an IStorageItem or IStorageFolder when you are running in an [AppContainer](https://docs.microsoft.com/en-us/windows/win32/secauthz/appcontainer-isolation). When running in an AppContainer the application has very limited file access rights. Things like picture and document folders require the native file handle to be proxied via RuntimeBroker.exe as the application process itself doesn't have rights to most files. This API gets the proxied handle. If accessing user data folders or using a file picker, this API should always be used to create `FileStream`. Using a path will often throw `AccessDeniedException` in these cases.

@mairaw I don't know how to write this in a more user friendly way. :)

Reply:
Fyi: Universal Windows apps run in an AppContainer.

Reply:
Thanks for your suggested remark, @JeremyKuhne . Do you want me to substitute the original remark with yours, or do you want me to append it?